Index: ComputerScience2/src/box.java
===================================================================
diff --git a/ComputerScience2/src/box.java b/ComputerScience2/src/box.java
deleted file mode 100644
--- a/ComputerScience2/src/box.java	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
+++ /dev/null	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
@@ -1,6 +0,0 @@
-public class box {
-    private Object object;
-
-    public void set(Object object){ this.object = object;}
-    public Object get() { return object;}
-}
Index: ComputerScience4/ComputerScienceFour.iml
===================================================================
diff --git a/ComputerScience4/ComputerScienceFour.iml b/ComputerScience4/ComputerScienceFour.iml
deleted file mode 100644
--- a/ComputerScience4/ComputerScienceFour.iml	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
+++ /dev/null	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: ComputerScience4/src/Main.java
===================================================================
diff --git a/ComputerScience4/src/Main.java b/ComputerScience4/src/Main.java
deleted file mode 100644
--- a/ComputerScience4/src/Main.java	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
+++ /dev/null	(revision 81d80d698d77194fc11fed8ae469d98daf2889a7)
@@ -1,5 +0,0 @@
-public class Main {
-    public static void main(String[] args) {
-
-        }
-}
\ No newline at end of file
